HISTORY.md

```diff
@@ -1,3 +1,19 @@
+# 5.0 / 2020-03-17
+
+This is a major new release that substantially cleans up and improves the codebase. It also introduces many new features, bugfixes, and removing major limitations. See [here](https://github.com/gollum/gollum/wiki/5.0-release-notes) for a list of changes.
```

Comment:
Maybe remove `substantially cleans up and improves` and just say `release that introduces many new features, [...]`. Also `removing` should be `removes`.